lib/commands/upload.js

```diff
@@ -48,6 +48,12 @@ function sendToGoogle(files, id) {
         }
       };
 
+      if (supportsTeamDrives) {
+        console.log('teamdrive=true')
```

Comment:
How about editing the message that shows up when uploading (something like "Pushing back up to Google Drive (target project is inside a Team Drive)..."?